nvdaHelper/vbufBackends/mshtml/node.cpp

```diff
@@ -124,15 +124,7 @@ class CDispatchChangeSink : public IDispatch {
 		if(dispIdMember==DISPID_EVMETH_ONPROPERTYCHANGE||dispIdMember==DISPID_EVMETH_ONLOAD) {
 			this->storageNode->backend->invalidateSubtree(this->storageNode);
 			// Force the update to happen with no delay if we happen to be in a live region
-			if (
-				this->storageNode->ariaLiveNode
-				&& this->storageNode->ariaLiveNode != this->storageNode
-				&& !this->storageNode->ariaLiveIsBusy
-				&& (
-					this->storageNode->ariaLiveIsTextRelevant
-					|| this->storageNode->ariaLiveIsAdditionsRelevant
-				)
-			) {
+			if(this->storageNode->ariaLiveNode&&this->storageNode->ariaLiveNode!=this->storageNode&&!this->storageNode->ariaLiveIsBusy&&(this->storageNode->ariaLiveIsTextRelevant||this->storageNode->ariaLiveIsAdditionsRelevant)) {

```

Comment:
I think that it's better to Split long if statements onto several lines. Do you have a reason to change this here and others places?